lib/mongo/error/transactions_not_supported.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+# rubocop:todo all
```

Comment:
For new files, I think we should probably apply the linter from the beginning.

Reply:
Good catch, thanks!